README.md

```diff
@@ -4,7 +4,7 @@ This is a lightweight fork of CosmosSDK. The current version of the forked code
 
 ## Making Changes to the Fork
 
-1. Open a PR against the current default branch (i.e. `dydx-fork-v0.47.0-alpha2`).
+1. Open a PR against the current default branch (i.e. `dydx-fork-v0.47.1`).
```

Comment:
From the latest update in `go.mod` in V4, it seems like we are using the [commit](f6e7e7a4448fd518b386b98dc44a359924f61216) on branch dydx-fork-v0.47.1.